### PR TITLE
[ETHOSN] Add support for transpose convolution

### PR DESCRIPTION
**Pull request by lhutton1**
Adds support for offloading transpose convolution with an optional bias to the NPU.

Co-authored-by: Samuel Panijel <samuel.panijel@arm.com>
Co-authored-by: Leo Blonk <leo.blonk@arm.com>

cc @leandron @Leo-arm @spanijel

### CODE REVIEW

File: python/tvm/relay/op/contrib/ethosn.py [hunk 1]
```diff
@@ -261,6 +271,13 @@ def check_mean(extract):
 
         return _ethosn.mean(extract)
 
+    def check_conv2d_transpose(extract):
+        """Check if mean is supported by Ethos-N."""
```
Comment (leandron) [inline python/tvm/relay/op/contrib/ethosn.py:275]: Docstring needs updating.

**Review by leandron — COMMENTED** [review]
Small nit. Apart from that, I'm happy for this to be merged.

**Comment by lhutton1** [comment]
@tvm-bot rerun

**Comment by lhutton1** [comment]
@tvm-bot rerun

**Review by leandron — APPROVED** [review]
LGTM. Thanks @lhutton1!

**Pull request merged**